### ISSUE TITLE
smooth not working properly on mac trackpad.

### ISSUE THREAD

**Issue by vikrantnegi** [inuyaksa/jquery.nicescroll#701]
I'm using Macbook pro and it seems that smooth scrolling work when I've mouse attached to the mac but scrolling doesn't work when I'm using the trackpad two fingers scrolling.

Is this a known issue or is there a workaround where we can disable smooth scrolling when using the trackpad?

Thanks.

**Comment by neonoxGER** [comment]
Not only trackpad, every MacBook (Pro) user suffers from this kind of script. I really don't understand why anybody needs such a script. Scrolling on OS X is perfect and this kind of JavaScript ruined that.

**Comment by vikrantnegi** [comment]
@neonoxGER You are right. I've tried like 10-15 types of smooth scrolling plugins but none of them seems to work on MacBooks when using the trackpad. 

But there are some sites that seem to be using smooth scrolling without even on MacBooks and I was wondering what scripts or plugin do they use to have this cross platform smooth scrolling effect.

**Comment by neonoxGER** [comment]
The thing with "smooth scrolling" is that only Windows PCs have a bad scrolling experience. No tablet, smartphone, or OS X device has this problem. Maybe this kind of plugin should only be active when a Windows device is used.

**Comment by kelp404** [comment]
Please automatically disable on OS X.
It is a real bad "smooth scrolling".

**Comment by gmonte** [comment]
In my [Dell Inspiron 5000](http://www.dell.com/pt-br/shop/notebooks-dell/inspiron-15-5000/spd/inspiron-15-5558-laptop) is the same problem. My trackpad don't working well (running win10 and ubuntu 14.06lts). A "if" OS X do not will working for me.
